sound/soc/intel/common/soc-acpi-intel-ptl-match.c

```diff
@@ -206,6 +215,15 @@ static const struct snd_soc_acpi_adr_device rt1320_1_group2_adr[] = {
 	}
 };
 
+static const struct snd_soc_acpi_adr_device rt1320_2_single_adr[] = {
+	{
+		.adr = 0x000230025d132000ull,
+		.num_endpoints = 1,
+		.endpoints = &single_endpoint,
```

Comment:
We should use group 1 as rt1320 works with rt712_vb's amp. IOW, rt712_vb and rt1320 are aggregated.